src/components/EmptyEventCard.tsx

```diff
@@ -0,0 +1,17 @@
+import { CalendarX } from '@phosphor-icons/react';
+
```

Comment:
This seems to be a copy of the no-events-card.tsx which already exists, can I please know why you introduced the same here @Kishan1835 ?

also was curious to know why an empty ClientThemeProvider has been added ?